docs/sources/flow/reference/components/prometheus.exporter.mongodb.md

```diff
@@ -29,6 +29,9 @@ Omitted fields take their default values.
 Name | Type | Description | Default | Required
 ---- | ---- | ----------- | ------- | --------
 `mongodb_uri` | `string` | MongoDB node connection URI. | | Yes
+`direct_connect` | `boolean` | Whether or not a direct connect should be made. Direct connections are not valid if multiple hosts are specified or an SRV URI is used. | true | No
+`discovering_mode` | `boolean` | Wheter or not to enable autodiscover collections. | false | No
+`tls_basic_auth_config_path` | `string` | Path to the file having Prometheus TLS config for basic auth. Only enable if you want to use TLS based authentication. Check [`tls_config`](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#tls_config) for reference on the file format | | No
```

Comment:
I'd move the second part of this line (the one starting with Check) below the table.